cypress/e2e/awx/access/organizations/organizations.cy.ts

```diff
@@ -204,3 +205,106 @@ describe('Organizations: Teams Tab', function () {
 
   it.skip('can edit a team from the teams tab inside of an Org', () => {});
 });
+
+cyLabel(['upstream'], () => {
```

Comment:
Is this label needed? This particular file gets excluded anyway.

Reply:
whoops! thank you!